starwars/src/Card.js

```diff
@@ -0,0 +1,19 @@
+import React, {Component} from 'react';
+import './Card.css';
+
+const Card = (props) => {
+  const char = props.char;
+  return (
+    <div className='card'>
+     <h1>{props.char.name}</h1> 
+     /* spent too much time playing with css */
```

Comment:
That's fine> Nicely done here. This is what we were looking for. A reusable component you could map over data and feed objects to it via props.